### PR TITLE
Support for ES 7.3.2

### PR DESCRIPTION
**Pull request by tclin1023**
Support for ES 7.3.2 and test with elasticsearch-oss-7.3.2-darwin-x86_64.tar.gz.
And adding UI support for adding additional field (opendistro_secuirty_roles) for internal user CRUD (https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/issues/111)

### CODE REVIEW

File: public/chrome/btn_logout/btn_logout.js [hunk 1]
```diff
@@ -29,23 +29,39 @@
...
+      let logoutButtonLabel = "Logout";
+      if (chromeInjected && chromeInjected.securityDynamic && chromeInjected.securityDynamic.user) {
+        if (!chromeInjected.securityDynamic.user.isAnonymousAuth) {
+          logoutButtonLabel = chromeInjected.securityDynamic.user.username;
+        } else {
+          logoutButtonLabel = "Login";
+        }
+      }
+      //console.log(ctrl.username);
+      ReactDOM.render(
```
Comment (mansi-amazon) [inline public/chrome/btn_logout/btn_logout.js]: hi, are we using react here?

File: public/chrome/btn_logout/btn_logout.js [hunk 2]
```diff
@@ -29,23 +29,39 @@
...
+        <div className="kbnGlobalNavLink__icon" style={{padding: "15px"}} onClick={onClick}>
+            <i className="fa fa-sign-out securityLogoutLink__icon-image"></i> {logoutButtonLabel}
+        </div>,
+        el
+      );
+      return () => ReactDOM.unmountComponentAtNode(el);
```
Comment (mansi-amazon) [inline public/chrome/btn_logout/btn_logout.js]: also here?

File: public/chrome/btn_logout/btn_logout.js [hunk 3]
```diff
@@ -29,23 +29,39 @@
  * permissions and limitations under the License.
  */
 
+import React from 'react';
```
Comment (mansi-amazon) [inline public/chrome/btn_logout/btn_logout.js]: Why do we need to import React?

File: public/chrome/btn_logout/btn_logout.js (hunk 1 above)
Comment (tclin1023) [reply public/chrome/btn_logout/btn_logout.js]: Yes, we use React render function which will receive a HTMLElement.

File: public/chrome/btn_logout/btn_logout.js (hunk 3 above)
Comment (tclin1023) [reply public/chrome/btn_logout/btn_logout.js]: Yes, we use React rendering framework here.

**Pull request merged**